cgkineo/adapt-list
# Ignore missing _graphic objects, update view and model filenames

Fixes #64 

### Fix
* Ignores missing `_graphic` objects instead of displaying an error
* Rename view and model files
* `Adapt.register` deprecation fix

### Testing
1. Set up a List component instance.
2. Remove (or do not add) the entire `_graphic` code block for a list item
3. Verify that there are no console errors relating to the missing `_graphic` object.
